src/components/hero2/hero.astro

```diff
@@ -0,0 +1,241 @@
+      viewBox="-40 -40 200 200"
+      ><text x="10" y="50" font-size="80" font-weight="bold" fill="#A0B7E9"
+        >&lt;/&gt;</text
+      ></svg
+    >
+    <svg
+      class="bg-icon bottom-[5%] left-[7%] rotate-[10deg] xl:block hidden"
+      viewBox="-40 -40 200 200"
+      ><text
+        x="10"
+        y="50"
+        font-size="80"
+        font-weight="bold"
+        fill="#A0B7E9"
+        set:html={"..."}
+      /></svg
+    >
+
+    <!-- Right side icons -->
+    <svg
+      class="bg-icon top-[5%] right-[17%] rotate-[30deg] xl:block hidden"
+      viewBox="-40 -40 200 200"
+      ><text x="10" y="50" font-size="80" font-weight="bold" fill="#A0B7E9"
+        >&lt;/&gt;</text
+      ></svg
+    >
+    <svg
+      class="bg-icon top-[30%] right-[6%] rotate-[15deg] xl:block hidden"
+      viewBox="-40 -40 200 200"
+      ><text x="10" y="50" font-size="80" font-weight="bold" fill="#A0B7E9"
+        >:=</text
+      ></svg
+    >
+    <svg
+      class="bg-icon bottom-[30%] right-[20%] rotate-[45deg] xl:block hidden"
+      viewBox="-40 -40 200 200"
+      ><text x="10" y="50" font-size="80" font-weight="bold" fill="#A0B7E9"
+        >&gt;&gt;&gt;</text
+      ></svg
+    >
+    <svg
+      class="bg-icon bottom-[5%] right-[8%] rotate-[33deg] xl:block hidden"
+      viewBox="-40 -40 200 200"
+      ><text x="10" y="50" font-size="80" font-weight="bold" fill="#A0B7E9"
+        >ƒ(x)</text
+      ></svg
+    >
+  </div>
+  <div class="relative z-10 text-center">
+    <div class="flex items-center space-x-3 p-10 md:mt-20">
+      <div class="w-25 h-25 flex items-center justify-center">
+        <HeaderLogo />
+      </div>
+      <h1 class="text-5xl md:text-9xl m-0 font-bold text-[#17223A]">
+        EuroPython
+      </h1>
+    </div>
+    <p class="max-w-2xl mx-10 md:mx-40 text-center text-lg text-gray-700 mt-2">
+      Discover the latest Python trends, learn from 180+ expert speakers, network with fellow developers, enjoy social events, and explore open spaces to spark new ideas.
+    </p>
+  </div>
+  <!-- First 2x1 Grid -->
+  <div class="grid grid-cols-1 md:grid-cols-2 gap-4 m-10">
+    <div>
+      <IconWithLabel icon="pin" label="Prague, Czech Republic" />
+    </div>
+    <div>
+      <IconWithLabel icon="calendar" label="14-20 July, 2025" />
+    </div>
+  </div>
+
+  <!-- Second 2x1 Grid with Action Buttons -->
+  <div class="grid grid-cols-1 md:grid-cols-2 gap-4 m-6">
+    <div>
+      <ButtonLink
+        secondary
+        url={action1}
+        className="mb-4 rounded-lg text-xl"
+        isExternal={true}
+      >
+        Register Now
+      </ButtonLink>
+    </div>
+    <div class="btn-outline">
+<a class="font-bold text-lg px-4 py-4 bg-button rounded-[10px] inline-block leading-4 hover:bg-button-hover not-prose outline-solid outline bg-transparent rounded-xlborder text-xl text-secondary outline-secondary text-black
+          " href="{action2}">
```

Comment:
```suggestion
          " href={action2}>
```
I don't think the attribute is set correctly with the quote. I was getting http://localhost:4321/%7Baction2%7D